sklearn/cross_decomposition/pls_.py

```diff
@@ -842,7 +864,7 @@ def fit_transform(self, X, y=None, **fit_params):
             Training vectors, where n_samples in the number of samples and
             p is the number of predictors.
 
-        Y : array-like of response, shape = [n_samples, q], optional
+        y : array-like of response, shape = [n_samples, q], optional
```

Comment:
No. Do you want me to capitalise it?

Also why is there the **fit_params arg if the `fit`method only takes X and Y? Should we remove this argument?